assets/ts/schedule/components/line-diagram/OtherStopList.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactElement } from "react";
+import { RouteStop, StopTree } from "../__schedule";
+import StopCard from "./StopCard";
+import { Alert } from "../../../__v3api";
+
+interface Props {
+  alerts: Alert[];
+  handleStopClick: (stop: RouteStop) => void;
+  otherRouteStops: RouteStop[];
+  searchQuery?: string;
+  stopTree: StopTree | null;
+}
+
+const stopCountWords = (stopCount: number): string =>
+  stopCount === 1 ? `1 Stop` : `${stopCount} Stops`;
+
+const OtherStopList = ({
+  alerts,
+  handleStopClick,
+  otherRouteStops,
+  searchQuery,
+  stopTree
+}: Props): ReactElement<HTMLElement> =>
+  otherRouteStops.length > 0 ? (
+    <>
+      <details className="group/other-stops">
+        <summary className="flex justify-between w-full bg-charcoal-90 border-[1px] group-open/other-stops:border-b-0 border-charcoal-80 cursor-pointer p-3 font-medium">
+          <span className="text-black">
+            {stopCountWords(otherRouteStops.length)} Served By Other Trips On
+            This Route
+          </span>
+          <span className="text-action-primary group-open/other-stops:hidden">
+            Show
+          </span>
+          <span className="text-action-primary hidden group-open/other-stops:inline">
+            Hide
+          </span>
+        </summary>
+        {otherRouteStops.map(stop => (
+          <StopCard
+            alerts={alerts}
+            key={stop.id}
+            noLineDiagram
+            onClick={handleStopClick}
+            routeStopList={otherRouteStops}
+            searchQuery={searchQuery}
+            stopId={stop.id}
+            stopTree={stopTree}
+          />
+        ))}
+      </details>
+    </>
+  ) : (
+    <></>
+  );
```

Comment:
This is such a nit: This can return nothing (`null`) instead of an empty `<React.Fragment/>` which still needs to be created...

Reply:
[I convinced Typescript to forgive me](https://github.com/mbta/dotcom/pull/2454/commits/ca262026a20b31686b6d920aa0a72caf95eca63e)